VCFv4.3.tex

```diff
@@ -577,43 +591,113 @@ \section{INFO keys used for structural variants}
 
 \footnotesize
 \begin{verbatim}
-##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
+##INFO=<ID=SVLEN,Number=A,Type=Integer,Description="Difference in length between REF and ALT alleles">
 \end{verbatim}
 \normalsize
-One value for each ALT allele. Longer ALT alleles (e.g.\ insertions) have positive values, shorter ALT alleles (e.g.\ deletions) have negative values.
+
+One value for each ALT allele.
+Longer ALT alleles (e.g.\ insertions) have positive values, shorter ALT alleles (e.g.\ deletions) have negative values.
+Inversion events should have $SVLEN=0$ since the total length of the allele matches the reference.
+The missing value should be used for inter-chromosomal $BND$ records.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=CIPOS,Number=.,Type=Integer,Description="Confidence interval around POS for imprecise variants">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be twice the number of alt allele (Number=2A).
```

Comment:
There are a few typos in the PR of the form ".. the number of alt allele" -- it should be ".. the number of alt alleles"